contracts/recovery-contracts/RecoveryWallet.sol

```diff
@@ -0,0 +1,66 @@
+pragma solidity ^0.4.18;
+import "./RecoveryForwarder.sol";
+import "./CloneFactory.sol";
+/**
+ *
+ * WalletSimple
+ * ============
+ *
+ * Basic singleSig wallet designed to recover funds.
+ *
+ */
+contract RecoveryWallet is CloneFactory {
+
+  // Public fields
+  address public signer;
+  address public forwarderImplementationAddress;
+
+  function RecoveryWallet (address _signer, address _forwarderImplementationAddress) public {
+    signer = _signer;
+    forwarderImplementationAddress = _forwarderImplementationAddress;
+  }
+
+  /**
+   * Modifier that will execute internal code block only if the sender is an authorized signer on this wallet
+   */
+  modifier onlySigner {
+    if (signer != msg.sender) {
+      revert();
+    }
+    _;
+  }
+
+ /**
+   * Gets called when a transaction is received without calling a method
+   */
+  function() public payable {
+  }
+
+  /**
+   * Create a new contract that forwards funds to this contract
+   */
+  function createForwarder() external {
```

Comment:
maybe worth having a batch create forwarder function? I know some wallets have bajillions of addresses

Reply:
yeah makes sense